### ISSUE TITLE
Generator/Slicer not working

### ISSUE THREAD

**Issue by quintindunn** [quintindunn/StringArtRobot#5]
Generator/Slicer needs to move the arm to the forwards position prior to travel to next pin, after final loop around it should stay in the forward position to keep tension on the thread.

**Comment by quintindunn** [comment]
I think I've worked out a sequence to loop around a given pin:

Variables:
`fw -> forward angle (arm)`
`bw -> backward angle (arm)`
`d1 -> Clockwise (CW)`
`d2 -> Counter-Clockwise (CCW)`
`spacing = 360/pin_count`

Sequence:
1.) Rotate to the pin
2.) Rotate `d1` `spacing/2`deg
3.) Arm from `fw`deg -> `bw`deg
4.) Rotate `d2` `spacing`deg
5.) Arm from `bw`deg -> `fw`deg
6.) Rotate `d1` `spacing`deg
7.) Arm from `fw`deg -> `bw`deg
8.) Rotate `d2` `spacing`deg
9.) Arm from `bw`deg to `fw`deg

Not able to test this right now as I'm printing a new design for the stepper mount, but will implement this on a generator branch.

**Comment by quintindunn** [comment]
Complete rework of how the motor interprets rotational commands, moving from relative positioning to absolute position relative to the zero'd position of the motor. (#6, #7)

This seems to be working as I anticipated now even without real world testing through simulation from my [StringArtRobotSimulator](https://github.com/quintindunn/StringArtRobotSimulator) project. Hopefully any further tuning can be conducted strictly through the parameters I added to the generator, likely specifically the Table sleep time (`tbsp`), which is the time in milliseconds between the motor finishing a movement and the next command to be executed in the sequence.
